--- orig/src/ehht.h
+++ tidy/src/ehht.h
@@ -19,9 +19,11 @@
 
 /* a simple hash-table */
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include <stddef.h>		/* size_t */
 
@@ -97,8 +99,10 @@
 /* destructor */
 void ehht_free(struct ehht_s *table);
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* EHHT_H */
--- orig/tests/echeck.h
+++ tidy/tests/echeck.h
@@ -19,9 +19,11 @@
 #ifndef ECHECK_H
 #define ECHECK_H
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include <stdlib.h>		/* for size_t */
 #include <stdio.h>		/* for FILE */
@@ -281,8 +283,10 @@
 #define check_status(val)\
 	echeck_status_m(stderr, ECHECK_FUNC, __FILE__, __LINE__, val, NULL)
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* ECHECK_H */
--- orig/tests/ehht-report.h
+++ tidy/tests/ehht-report.h
@@ -17,9 +17,11 @@
 #ifndef EHHT_REPORT_H
 #define EHHT_REPORT_H
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include <ehht.h>
 
@@ -48,8 +50,10 @@
 	return i;
 }
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #endif /* EHHT_REPORT_H */
